### ISSUE TITLE
Change seasonality of "D" to 7, and 52 for "W"

### ISSUE THREAD

**Issue by AaronSpieler** [awslabs/gluon-ts#975]
## Description
- currently we have `DEFAULT_SEASONALITIES = {"H": 24, "D": 1, "W": 1, "M": 12, "B": 5}`
- however, having a seasonality of  1 has no real benefit to my knowledge  (please correct me if I'm mistaken)
  - I base my knowledge on NBEATSEstimator and Naive2Predictor
- therefore I suggest seasonality of 7 for Daily data "D" 
  - to me this makes sense since this would capture the seasonality of workdays and weekends and in general the work week
- additionally we could think of a seasonality of 52 (or 4) for weekly data "W"

## Examples

- Wavenet:
```
                {
                    "H": 7 * 24,
                    "D": 7,
                    "W": 52,
                    "M": 12,
                    "B": 7 * 5,
                    "min": 24 * 60,
                },
```

- DeepState:
```
FREQ_LONGEST_PERIOD_DICT = {
    "M": 12,  # yearly seasonality
    "W-SUN": 52,  # yearly seasonality
    "D": 31,  # monthly seasonality
    "B": 22,  # monthly seasonality
    "H": 168,  # weekly seasonality
    "T": 1440,  # daily seasonality
}
```

**Comment by kaijennissen** [comment]
Any plans to change this in the near future? Because I recently worked with the `SeasonalNaivePredictor` and daily observations and was surprised that I had to manually set `season_length=7` despite the call to `get_seasonality(freq)` in the `__init__`-method.

**Comment by jaheba** [comment]
I agree that we should have a daily seasonality of `7`.  Are there cases that we are missing, i.e. why was `1` chosen in the first place?

However, my feeling is that there is not such a fitting seasonality for weekly-frequency.
